package.json

```diff
@@ -17,23 +17,29 @@
     "url": "shelf.io"
   },
   "files": [
-    "environment.js",
+    "lib/environment.js",
     "jest-preset.js",
-    "setup.js",
-    "teardown.js",
-    "wait-for-localhost.js"
+    "lib/setup.js",
+    "lib/teardown.js",
+    "lib/index.d.ts",
+    "lib/utils/wait-for-localhost.js"
```

Comment:
Also, can we specify the full `lib/` directory here?

Reply:
It looks like this now
<img width="321" alt="image" src="https://user-images.githubusercontent.com/19535641/183654438-84ca4ff9-bafa-4ec6-81d8-7fd0fed8523b.png">

Reply:
I guess all the files, apart from *d.ts are specified

Reply:
But we need `d.ts` files in our package too.

Reply:
TS support would not work without it.

Reply:
https://docs.npmjs.com/cli/v8/configuring-npm/package-json#files

We can use patterns too

>  File patterns follow a similar syntax to .gitignore, but reversed: including a file, directory, or glob pattern (*, **/*, and such) will make it so that the file is included in the tarball when it's packed

Reply:
Okay, nice, but do we need to use patterns here?

Reply:
We can, it's easier to specify the full lib folder with pattern - one line,  instead of writing every single file one by one

Reply:
I mean, why do we need to specify a pattern instead of the exact directory `lib/`?

Reply:
Yes, we can do it without pattern

Reply:
Changed